app/helpers/related_urls_helper.rb

```diff
@@ -0,0 +1,654 @@
+module RelatedUrlsHelper
```

Comment:
I think there is some redundancy in these systems. I think that the content map + schema should be enough to drive related URLs without the need to use the controlled selects. I am unsure if we want to address this in our current tickets or write a ticket to investigate it later.

Reply:
I agree, and I think we should have a future ticket to do that. I will also add a note in 2223 to do that.